docs/docs/train-tutorial.md

```diff
@@ -29,7 +29,9 @@ $ zamba train --data-dir example_vids/ --labels example_labels.csv
 To run `zamba train` in the command line, you must specify both `--data-directory` and `--labels`.
 
 * **`--data-dir PATH`:** Path to the folder containing your labeled videos. `zamba` will generate predictions for videos in the top level directory and in any nested folders.
-* **`--labels PATH`:** Path to a CSV containing the video labels to use as ground truth during training. There must be columns for both filepath and label. Filepaths should be either absolute paths or relative to `data-dir`. Optionally, there can also be columns for `split` (`train`, `val`, or `holdout`) and `site`. If your labels file does not have a column for `split`, you can alternately use the `split_proportions` argument.
+* **`--labels PATH`:** Path to a CSV containing the video labels to use as ground truth during training. There must be columns for both filepath and label. Filepaths should be either absolute paths or relative to `data-dir`. Optionally, there can also be columns for `split` (which can have one of the three values for each row: `train`, `val`, or `holdout`) and `site` (which can contain any string identifying the location of the camera). If your labels file does not have a column for `split`, you can alternately use the `split_proportions` argument (see below).
```

Comment:
Would remove "(see below)" since i don't think there is a `split_proportions` elsewhere the page (or link to the page where this is explained)